From f69b74606b6234049d11e048b6d40c4408c859a5 Mon Sep 17 00:00:00 2001
From: Hong-Mei Li <a21834@motorola.com>
Date: Thu, 24 Sep 2015 01:41:52 -0700
Subject: [PATCH 184/959] IKSWM-4458 motoconfig:8952: Enable pstore fs

Enable pstore to get system logs, currently, we enabled:
last_kmsg, panic and annotation

Change-Id: Ib92728401e903358d6782e52cd6c774a4a3cbe87
Signed-off-by: Hong-Mei Li <a21834@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 6df2301857b..0ca2ad77812 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -26,3 +26,7 @@ CONFIG_MOT_UTAG=y
 CONFIG_PROC_DEVICETREE=y
 CONFIG_DIAG_OVER_TTY=y
 CONFIG_PANIC_ON_RT_THROTTLING=y
+CONFIG_PSTORE=y
+CONFIG_PSTORE_CONSOLE=y
+CONFIG_PSTORE_RAM=y
+CONFIG_PSTORE_RAM_ANNOTATION_APPEND=y
-- 
2.11.0

